packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -80,16 +96,91 @@ or you can use the `IStage.addAction()` method to mutate an existing Stage:
 sourceStage.addAction(someAction);
 ```
 
+### Cross-account CodePipelines
+
+Cross-account Pipeline actions require that the Pipeline has *not* been
+created with `crossAccountKeys: false`.
+
+To perform actions in a different account than your Pipeline is in, most
+actions accept a resource that is in a different account (either created
```

Comment:
Sentence not clear, specifically what you mean by "accept a resource". What kind of resource?

Rephrase.

Reply:
Please suggest something better. It can be a Bucket, a CodeDeploy DeploymentGroup, an ECS Cluster. What is a good enough generic term that is not too generic in your opinion?